h2o-extensions/xgboost/src/main/java/hex/tree/xgboost/XGBoost.java

```diff
@@ -56,6 +56,10 @@
   public XGBoost(boolean startup_once) { super(new XGBoostModel.XGBoostParameters(),startup_once); }
   public boolean isSupervised(){return true;}
 
+  @Override protected int nModelsInParallel(int folds) {
+    return nModelsInParallel(folds, 2);
```

Comment:
@michalkurka : at that time, the default parallelization for XGBoost was 2, same as GBM. Should I changed it to 1? For info, seems to work fine with 2.

Reply:
@seb-h2o Does this mean that only 2 models are trained in parallel even if `nfolds = 5`?  I think we should really review the defaults for CV parallelization based on the size of the data vs size of the H2O cluster, but that's a conversation for another day!

Reply:
that's correct, and afair only GBM (and in this PR, XGB) have parallelization activated for large data. It's also always activated for most algos when data are very small, like < 1MB.
I also already added a comment in ModelBuilder that goes in the same direction: we could really improve this heuristic based on cluster size, nthreads, data size... maybe we should just create a ticket for this.

Reply:
@ledell created https://0xdata.atlassian.net/browse/PUBDEV-6312 for future reference

Reply:
@seb-h2o Thanks.  We have a simplified ticket here (based on data size only), I'll link them in JIRA: https://0xdata.atlassian.net/browse/PUBDEV-5783